client/src/api/utils.ts

```diff
@@ -0,0 +1,69 @@
+let windowsSystem: boolean | undefined;
+
+export function throwErrorAsString(e: any) {
+  let stringErr: string;
+  if (e.stderr !== undefined) {
+    stringErr = e.stderr;
+  } else {
+    stringErr = e.toString();
+  }
+  throw stringErr;
+}
+
+/**
+ * Executes a command on the host machine. Results and outputs are returned when the process is closed.
+ * @param unixCmd command, binary or script to run on macOS and Linux machines.
+ * @param windowsCmd command, binary or script to run on Windows machines.
+ * @param args
+ */
+export async function execOnHost(unixCmd: string, windowsCmd: string, args?: string[]): Promise<any> {
+  if (await isWindows()) {
+    return window.ddClient.extension.host.cli.exec(windowsCmd, args);
+  }
+  return window.ddClient.extension.host.cli.exec(unixCmd, args);
+}
+
+/**
+ * Executes a command on the host machine, and streams the output, even before the process is closed.
+ * @param unixCmd command, binary or script to run on macOS and Linux machines.
+ * @param windowsCmd command, binary or script to run on Windows machines.
+ * @param args
+ * @param options an ExecStreamOptions object, as described in Docker Desktop Extensions docs.
+ */
+export async function execOnHostAndStreamResult(unixCmd: string, windowsCmd: string, args: string[], options: any): Promise<any> {
+  if (await isWindows()) {
+    return window.ddClient.extension.host.cli.exec(windowsCmd, args, options);
+  }
+  return window.ddClient.extension.host.cli.exec(unixCmd, args, options);
+}
+
+export async function isWindows(): Promise<boolean> {
+  if (windowsSystem === undefined) {
+    windowsSystem = navigator.platform.startsWith('Win');
+  }
+  return windowsSystem;
+}
+
+export class Versions {
+  xrayVersion?: string;
+  jfrogCliVersion?: string;
+}
+
+/**
+ * Gets the versions of JFrog CLI (that's used by the extension) and JFrog Xray.
+ */
+export async function getVersions(): Promise<Versions> {
```

Comment:
Does this function return two versions? Which version is the first and which is the second? I suggest considering to modify the return value in a way that will make it clearer, or at least explain this as part of the description.